openhands/events/stream.py

```diff
@@ -29,24 +30,17 @@ def session_exists(sid: str, file_store: FileStore) -> bool:
         return False
 
 
+@dataclass
 class EventStream:
     sid: str
     file_store: FileStore
     # For each subscriber ID, there is a stack of callback functions - useful
     # when there are agent delegates
-    _subscribers: dict[str, list[Callable]]
-    _cur_id: int
-    _lock: threading.Lock
-
-    def __init__(self, sid: str, file_store: FileStore):
-        self.sid = sid
-        self.file_store = file_store
-        self._subscribers = {}
-        self._cur_id = 0
-        self._lock = threading.Lock()
-        self._reinitialize_from_file_store()
+    _subscribers: Dict[str, list[Callable]] = field(default_factory=dict)
```

Comment:
```suggestion
    _subscribers: dict[str, list[callable]] = field(default_factory=dict)
```

Reply:
Unfortunately ruff complains about the use of `callable` as a type hint, so I can't do this one for now